stdlib/math/math.flux

```diff
@@ -0,0 +1,198 @@
+package math
+
+// builtin constants
+builtin pi
+builtin e
+builtin phi
+builtin sqrt2
+builtin sqrte
+builtin sqrtpi
+builtin sqrtphi
+builtin ln2
+builtin log2e
+builtin ln10
+builtin log10e
+builtin maxfloat
+builtin maxint
+builtin minint
+builtin maxuint
+
+//missing builtin values [y1 sqrte roundtoeven nextafter32 sqrtphi sqrtpi copysign ceil], extra builtin values [Ceil sqrtPhi sqrtPi sqrtE roundToEven copySign]
+// builtin functions
+builtin abs
+builtin acos
+builtin acosh
+builtin asin
+builtin asinh
+builtin atan
+builtin atan2
+builtin atanh
+builtin cbrt
+builtin ceil
+builtin copysign
+builtin cos
+builtin cosh
+builtin dim
+builtin erf
+builtin erfc
+builtin erfcinv
+builtin erfinv
+builtin exp
+builtin exp2
+builtin expm1
+builtin float64bits
+builtin float64frombits
+builtin floor
+builtin frexp
+builtin gamma
+builtin hypot
+builtin ilogb
+builtin m_inf
+builtin isInf
+builtin isNaN
+builtin j0
+builtin j1
+builtin jn
+builtin ldexp
+builtin lgamma
+builtin log
+builtin log10
+builtin log1p
+builtin log2
+builtin logb
+builtin m_max
+builtin m_min
+builtin mod
+builtin modf
+builtin NaN
+builtin nextafter
+builtin pow
+builtin pow10
+builtin remainder
+builtin round
+builtin roundtoeven
+builtin signbit
+builtin sin
+builtin sincos
+builtin sinh
+builtin sqrt
+builtin tan
+builtin tanh
+builtin trunc
+builtin y0
+builtin y1
+builtin yn
+
+// Table transformations for trig functions
+sinValue = (tables=<-) => tables |> map(fn:(r) => sin(x:r._value))
+cosValue = (tables=<-) => tables |> map(fn:(r) => cos(x:r._value))
+tanValue = (tables=<-) => tables |> map(fn:(r) => tan(x:r._value))
+asinValue = (tables=<-) => tables |> map(fn:(r) => asin(x:r._value))
+acosValue = (tables=<-) => tables |> map(fn:(r) => acos(x:r._value))
+atanValue = (tables=<-) => tables |> map(fn:(r) => atan(x:r._value))
+
+// hack to simulate an imported math package
```

Comment:
why do we need this?

Reply:
I mean, why putting both in the prelude and making it available as an imported package?

Reply:
when we fix an issue involving visibility of package functions in calls to `map` and `filter`  this can go away.

Reply:
So you need it in the prelude to make map and filter see math functions?

Reply:
yes, and this would apply to any package, e.g., package `strings`  does not have this hack and therefore is currently not visible for `map` or `filter`.